src/pages/sc-cycle-counts/Forms/CycleCountsForm.js

```diff
@@ -212,7 +212,7 @@ export default function CycleCountsForm({ labels, maxAccess: access, setStore, s
     })()
   }, [])
 
-  const disable = formik.values.status === 3 || isClosed
+  const disable = isPosted || isClosed
```

Comment:
why do we need this? 
you can ask if this is better or just using isposted and is closed in our fields is enough